### ISSUE TITLE
Change storage of authorization token

### ISSUE THREAD

**Issue by scrishel** [Tiny-Hands/tinyhands-client#411]
Change the storage of the authorization token for better security.  The authorization token is currently maintained in localStorage.  Users remain logged in until they explicitly log out.  Even if the user exits the browser, the token remains.  Update the system to maintain the token in sessionStorage.

**Comment by benaduggan** [comment]
This is really only security through obscurity here though, because this doesn't invalidate the tokens. I'd recommend adding a expiration time of a week on the tokens instead. Personally I just get super annoyed when websites make me login every time I open a new browser window.

**Comment by scrishel** [comment]
I don't enjoy entering a password every time I open a new browser window either although LastPass makes it pretty painless.  We are storing personal sensitive information on this system (passport numbers, citizenship id numbers, victim's name and  phone numbers, etc).  Can you think of any system that stores that type of sensitive data that does not force a login when you enter a new browser window?

I will let Austin provide his feedback.  It was at his suggestion that this issue was created.

**Comment by amunn33** [comment]
@scrishel Yeah, that's my fault on the false alarm. I'm trying to figure out what the issue was that was described to me because our station in South Africa is using the same computer and I heard a second-hand account that they were unable to sign in with anyone else's account so they've just been using one person's account. I instantly thought it was because of the issue that I mentioned because I had seen behavior like that in the past, for both myself and others trying to logout/login, so now I'm wondering what was really happening there.

@benaduggan We don't actually have active generic border station users right now and the plan would be that when station staff start to enter forms directly, which is beginning to happen in South Africa, that they would have their own accounts.

**Comment by amunn33** [comment]
![GitHub](https://user-images.githubusercontent.com/7903547/57385055-6ea63280-71d1-11e9-9744-a62036f659ce.png)
Also, GitHub is having some issues working out these timezones!
